core/UrlHelper.php

```diff
@@ -231,7 +231,10 @@ public static function getPathAndQueryFromUrl($url)
         $parsedUrl = parse_url($url);
         $result = '';
         if (isset($parsedUrl['path'])) {
-            $result .= substr($parsedUrl['path'], 1);
+            if (substr($parsedUrl['path'], 0, 1) == '/') {
+                $parsedUrl['path'] = substr($parsedUrl['path'], 1);
```

Comment:
There was a bug if `$url` was eg `Others` it resulted before in `thers` see test.